distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/RemovePluginCommandTests.java

```diff
@@ -251,6 +252,15 @@ public void testMissingPluginName() throws Exception {
         assertEquals("plugin name is required", e.getMessage());
     }
 
+    public void testRemoveIngestGeoIp() {
+        final UserException e = expectThrows(UserException.class, () -> removePlugin("ingest-geoip", home, randomBoolean()));
+        assertThat(e.exitCode, equalTo(ExitCodes.OK));
+        assertThat(
+                e,
+                hasToString(Matchers.containsString(
+                        "ingest-geoip is no longer a plugin but instead a module packaged with this distribution of Elasticsearch")));
+    }
```

Comment:
Can you please add a test here for the case the plugin is installed and it still gets removed?

Reply:
I pushed 503b055722cc4f7d9cb20a7e18eaef3fe6eb1cad.